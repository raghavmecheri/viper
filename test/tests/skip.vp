int i = 1;
while (i < 10) {
    if (i == 5) {
        int x = 444;
        print("bad");
        skip;
    }
    print(i);
    i = i + 1;
}
