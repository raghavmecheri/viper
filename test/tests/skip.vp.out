0
1
2
3
4
bad
6
7
8
9

0
skipping
1
did not skip
2
did not skip
3
did not skip
4
skipping
5
did not skip
6
did not skip
7
did not skip
8
skipping
9
did not skip
10
did not skip
11
did not skip
12
skipping
13
did not skip
14
did not skip
15
did not skip
16
skipping
17
did not skip
18
did not skip
19
did not skip
20
skipping
21
did not skip
22
did not skip
23
did not skip
24
skipping
25
did not skip
26
did not skip
27
did not skip
28
skipping
29
did not skip
30
did not skip
31
did not skip
32
skipping
33
did not skip
34
did not skip
35
did not skip
36
skipping
37
did not skip
38
did not skip
39
did not skip
40
skipping
41
did not skip
42
did not skip
43
did not skip
44
skipping
45
did not skip
46
did not skip
47
did not skip
48
skipping
49
did not skip
50
did not skip
51
did not skip
52
did not skip
53
did not skip
54
did not skip
55
did not skip
56
did not skip
57
did not skip
58
did not skip
59
did not skip
60
did not skip
61
did not skip
62
did not skip
63
did not skip
64
did not skip
65
did not skip
66
did not skip
67
did not skip
68
did not skip
69
did not skip
70
did not skip
71
did not skip
72
did not skip
73
did not skip
74
did not skip
75
did not skip
76
did not skip
77
did not skip
78
did not skip
79
did not skip
80
did not skip
81
did not skip
82
did not skip
83
did not skip
84
did not skip
85
did not skip
86
did not skip
87
did not skip
88
did not skip
89
did not skip
90
did not skip
91
did not skip
92
did not skip
93
did not skip
94
did not skip
95
did not skip
96
did not skip
97
did not skip
98
did not skip
99
did not skip